tests/scripts/analyze_outcomes.py

```diff
@@ -214,6 +214,324 @@ def do_analyze_driver_vs_reference(results: Results, outcome_file, args):
             }
         }
     },
+    'analyze_driver_vs_reference_cipher_aead': {
+        'test_function': do_analyze_driver_vs_reference,
+        'args': {
+            'component_ref': 'test_psa_crypto_config_reference_cipher_aead',
+            'component_driver': 'test_psa_crypto_config_accel_cipher_aead',
+            # Ignore suites that are being accelerated
```

Comment:
This comment doesn't make sense to me: we don't accelerate suites. I guess the intention is to ignore suites that are specific to the built-in implementation of mechanisms that are accelerated in the driver component?

Reply:
Sorry, that was the idea but I phrased it very badly

Reply:
FWIW, in #8398 I went with "Modules replaced by drivers".